### PR TITLE
Make evaluation of BEP006 optional

### PR DESCRIPTION
**Pull request by sappelhoff**
This adds a new option to the command line: `bep006`, which is a boolean defaulting to `false`. If the bids-validator is called with the `--bep006` flag, BEP_006 will be evaluated. Else, the initial "quick test" will be failed.

image

### CODE REVIEW

**Comment by sappelhoff** [comment]
Note: I accidentally added `package-lock.json` that must be removed before a potential merge.

File: bin/bids-validator [hunk 1]
```diff
@@ -10,18 +10,18 @@ var argv = require('yargs')
...
+    .describe('ignoreWarnings', 'Disregard non-critical issues')
     .boolean('ignoreNiftiHeaders')
-    .describe('ignoreNiftiHeaders', 'disregard NIfTI header content during validation')
+    .describe('ignoreNiftiHeaders', 'Disregard NIfTI header content during validation')
     .boolean('verbose')
-    .describe('verbose', 'Log more extensive information about issues.')
+    .describe('verbose', 'Log more extensive information about issues')
     .boolean('json')
-    .describe('json', 'Output results as JSON.')
+    .describe('json', 'Output results as JSON')
```
Comment (sappelhoff) [inline bin/bids-validator:19]: Up to here, cosmetic changes for consistency (start description with capital letter, do not end with a dot)

**Comment by chrisgorgo** [comment]
Wouldn't this only work on pure EEG datasets? Datasets with other modalities will pass the quickTest and claim EEG files are valid BIDS even without --bep006 - or am I missing something?

**Comment by sappelhoff** [comment]
> Wouldn't this only work on pure EEG datasets? Datasets with other modalities will pass the quickTest and claim EEG files are valid BIDS even without --bep006 - or am I missing something?

right ... when inserting e.g., a `func` directory copied from an MRI example, the quick test is passed and the EEG is treated as if it was part of BIDS.

To fix this, I applied the same logic as above to the "full test" now, targeting the `isBIDS` function and passing `BIDS.options.bep006` further down. 

Now when testing that EEG repository where I added a single `func` directory for a subject, we get the following output:

image

**Comment by chrisgorgo** [comment]
great, thanks!

**Pull request merged**